src/config/theme/components/layout.ts

```diff
@@ -20,39 +20,50 @@ const layout = {
       justifyContent: 'space-between',
       alignItems: 'center',
     },
+    title: {
+      color: 'toolbarPrimary',
+    },
+    subtitle: {
+      color: 'toolbarSecondary',
+    },
   },
   nav: {
     container: {
       mt: [3, 0],
       display: 'flex',
       justifyContent: 'space-around',
+      alignItems: 'center',
       alignSelf: 'stretch',
     },
     item: {
-      color: 'textSecondary',
+      alignItems: 'center',
+      color: 'toolbarSecondary',
+      display: 'flex',
+      flexDirection: 'column',
       textDecoration: 'none',
       fontSize: [1, 3],
       fontWeight: 'bold',
+      borderBottom: '2px solid',
+      borderBottomColor: 'toolbar',
       ml: [0, 4],
       '&:hover': {
         color: 'text',
       },
     },
     itemActive: {
       variant: 'components.layout.nav.item',
-      color: 'text',
-      borderBottom: '2px solid',
-      borderBottomColor: 'primary',
+      color: 'toolbarPrimary',
+      borderBottomColor: 'toolbarPrimary',
```

Comment:
Let's not change this. And instead just set the border to `toolbarSecondary` when the item is active.